app/views/Main.js

```diff
@@ -48,6 +49,22 @@ const Main = () => {
     );
   };
 
+  const QRIcon = () => {
```

Comment:
I would recommend shifting the button to the Settings screen instead. This allows providing some context in the subtitle about the feature as well and also we are keeping all standalone features linkable through Settings screen.